github_pr.py

```diff
@@ -353,12 +381,13 @@ def main():
 
     github-pr merge -r dataxu/test_repo -n 17
 
-    github-pr merge -r dataxu/test_repo -n 17 --mergecondition-owner-cannot-ship --mergecondition-approved-mergers-file=.approved-mergers-file
+    github-pr merge -r dataxu/test_repo -n 17 --condition-non-owner-merger --condition-approved-mergers-file=.approved-mergers-file
```

Comment:
this example is passing two `--condition...` arguments, but we only iterate on conditions and call with them singly in the jenkins job - just wanted to confirm we aren't possibly masking any false positives?